File: C & C++ PRACTICE CODES/Nested for loop Basic-2/code8.c

#include <stdio.h>

void main (){

	int row ;
	printf("enter no of row: ");
	scanf("%d",&row);

	for(int i =0; i < row ; i++){

		for(int a = 0;a < row ; a++){

			printf("\tA%d\t",a+ 1);

			
		}
		printf("\n\n");
	}


}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/8/code8.java
import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("enter no. of rows");
		int row = s.nextInt();
		int temp = 1;


		for(int i = 0; i < row ; i++){
			for(int j = 0;j <row ;j++){
                                                                  
				if(temp % 2==0){

					System.out.print((temp)+"		");

				}else{
					System.out.print((temp*temp)+"		");
				}

				temp++;
			}
			
			
			System.out.println("\n");
		}
	}
}
